.github/workflows/version.yml

```diff
@@ -0,0 +1,20 @@
+on:
+  release:
+    types: [published]
+jobs:
+  add-dev-to-version:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3 
+    - name: Append version with dev
+      run: sed -i '/20[0-9][0-9]\.[0-9][0-9]/ s/]/-dev]/' configure.ac 
+    - name: Create pull request
+      uses: peter-evans/create-pull-request@v4 
+      with:
+        base: main                   # creates a new branch off of main
+        branch: add-dev-post-release # name of the created branch
+        branch-suffix: timestamp     # add a timestamp to branch name
+        delete-branch: true          # delete afer merge 
+        title: Append dev to version number post-release
+        body: automated change, adds '-dev' to the version number upon releases
+        reviewers: rem1776
```

Comment:
Is the `reviewers` required?  Will rem1776 be the only required reviewer?  This could cause a potentially dangerous situation if you're on leave or away at the time of a release and a backup person is required.

Reply:
i think it adds the other reviewers based on codeowners like it does for other prs, but i mean it's always the same changes so i'll just add you and rusty to that list

Reply:
Can you get rid of reviewers entirely?  Then it should just add the 3 of us, right?

Reply:
that works too